### PR TITLE
feat: add basic analysis sample project

### PR DESCRIPTION
**Pull request by Itsysho**
# Description
add basic analysis sample project
* add basic analysis document
* add basic analysis sample project

### CODE REVIEW

File: basic-analysis/basic-analysis.html [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    @media screen and (orientation: portrait) {
+      .device-ios .player { 
+        height: calc(100vh - 114px);
+      }
+    }
+  </style>
+</head>
+<body>
+  <div id='my-player'></div>
+  <script src="https://unpkg.com/@blendvision/player@2.10.0-canary.0"></script>
```
Comment (Itsysho) [inline basic-analysis/basic-analysis.html]: Need to replace to formal release version `https://unpkg.com/@blendvision/player@2.10.0`

File: basic-analysis/README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+- Analytics Field Definition
+    - `analytics.token`: The field is required, this appears to be a token or authentication key for analytics purposes.
+
+    - `analytics.sessionId`: The field is optional, this represents a session identifier, likely used to track user sessions or interactions with the application.
+
+    - `analytics.resourceId`: The field is optional, this could be an identifier for a specific resource or source within your application, used for tracking or categorization.
+
+    - `analytics.resourceType`: The field is optional, this might indicate the type or category of the resource mentioned above.
+
+    - `analytics.customData`: The field is optional, this seems to be a placeholder for custom data that you can include in your analytics events, allowing you to send additional information as needed.
```
Comment (dk00) [inline basic-analysis/README.md]: For latest version, `analytics.<customProperty>` is supported instead, for example `analytics.my_log_prop` will add property `my_log_prop`.

**Review by dk00 — CHANGES_REQUESTED** [review]

File: basic-analysis/basic-analysis.html (hunk 1 above)
Comment (Itsysho) [reply basic-analysis/basic-analysis.html]: replace in [0b87f5b](https://github.com/BlendVision/web-player-samples/pull/4/commits/0b87f5b2aaed66aec20ff6298d6a826c329c0a0c)

File: basic-analysis/README.md (hunk 2 above)
Comment (Itsysho) [reply basic-analysis/README.md]: fix in [ad0a9e7](https://github.com/BlendVision/web-player-samples/pull/4/commits/ad0a9e774dc001edd6cf567539f65b53e822408c)

**Review by dk00 — APPROVED** [review]

**Pull request merged**